service/handler/quote_test.go

```diff
@@ -0,0 +1,89 @@
+package handler
+
+import (
+	"io/ioutil"
+	"net/http"
+	"net/http/httptest"
+	"strings"
+	"testing"
+
+	"github.com/etyberick/golang-bootcamp-2020/entity"
+	"github.com/etyberick/golang-bootcamp-2020/service/repository"
+	"github.com/etyberick/golang-bootcamp-2020/usecase/interactor"
+)
+
+const quoteRoute = "/v0/quote"
+
+// Normal update test
+func TestUpdate(t *testing.T) {
+	// Mock a new request
+	req, err := http.NewRequest("POST", "/v0/quote", nil)
```

Comment:
Should we use `httptest` here?

Reply:
This was resolved because that package was used later below.